README.md

```diff
@@ -1,56 +1,62 @@
 [![REUSE status](https://api.reuse.software/badge/github.com/SAP/ai-sdk-js)](https://api.reuse.software/info/github.com/SAP/ai-sdk-js)
 [![Fosstars security rating](https://github.com/SAP/ai-sdk-js/blob/fosstars/fosstars_badge.svg)](https://github.com/SAP/ai-sdk-js/blob/fosstars/fosstars_report.md)
 
-# ai-sdk-js
+# SAP AI SDK
 
-## Disclaimer
+Integrate chat completion into your business applications with SAP Cloud SDK for GenAI Hub. Leverage the Generative AI Hub of SAP AI Core to make use of templating, grounding, data masking, content filtering and more. Setup your SAP AI Core instance with SAP Cloud SDK for AI Core.
+
+## Disclaimer ⚠️
 
 This project is currently in an experimental state. All functionality and naming is subject to change. Use at your own discretion.
 
-## About this project
+## Table of Contents
 
-Integrate chat completion into your business applications with SAP Cloud SDK for GenAI Hub. Leverage the Generative AI Hub of SAP AI Core to make use of templating, grounding, data masking, content filtering and more. Setup your SAP AI Core instance with SAP Cloud SDK for AI Core.
+- [Packages](#packages)
+  - [@sap-ai-sdk/ai-core](#sap-ai-sdk-ai-core)
+  - [@sap-ai-sdk/gen-ai-hub](#sap-ai-sdk-gen-ai-hub)
+- [Sample Project](#sample-project)
+- [Support, Feedback](#support-feedback-contribution)
+- [Contribute](#contribute)
+- [Links](#links)
+- [License](#license)
 
-## Requirements and Setup
+## Packages
 
-_Insert a short description what is required to get your project running..._
+This project publishes multiple packages and is managed using [pnpm](https://pnpm.io/)
 
-## Running tests
+### @sap-ai-sdk/ai-core
 
-If you're using the Jest Runner extension in Visual Studio Code, you'll need to add the following settings to your `settings.json` file before running the tests from VSCode:
+This package provides tools to manage your scenarios and workflows in SAP AI Core.
 
-```
- "jestrunner.debugOptions": {
-    "runtimeArgs": ["--experimental-vm-modules"]
-  },
-  "jestrunner.jestCommand": "NODE_OPTIONS=--experimental-vm-modules node 'node_modules/jest/bin/jest.js'",
-```
+- Streamline data preprocessing and model training pipelines
+- Execute batch inference jobs
+- Deploy inference endpoints for your trained models
+- Register custom Docker registries, sync AI content from your own git repositories, and register your own object storage for training data and model artifacts
 
-You can run our tests either with:
+To install the SAP AI Core in your project, run:
 ```
-pnpm test:unit
-pnpm test:type
+$ npm install @sap-ai-sdk/ai-core
 ```
 
```

Comment:
[suggestion] Also link to the respective section's dedicated README's for usage details.